src/interval.jl

```diff
@@ -191,3 +199,11 @@ function detecthlinearity!(::Interval) end
 function detectvlinearity!(::Interval) end
 function removehredundancy!(::Interval) end
 function removevredundancy!(::Interval) end
+
+function sethrep!(p::Interval{T, AT}, h::HRep) where {T, AT}
+    hnew, v, volume = _hinterval(h, AT)
+    hrep = hnew
+    vrep = v
+    length = volume
+    return p
+end    
```

Comment:
Do the three lines in the middle have any effect? If not, then this method does not modify `p` (as probably intended).

Reply:
Yes you're right thank you! I'll change that